repos/spack_repo/builtin/packages/libint/package.py

```diff
@@ -44,46 +36,51 @@ class Libint(AutotoolsPackage):
     version("1.1.6", sha256="f201b0c621df678cfe8bdf3990796b8976ff194aba357ae398f2f29b0e2985a6")
     version("1.1.5", sha256="ec8cd4a4ba1e1a98230165210c293632372f0e573acd878ed62e5ec6f8b6174b")
 
-    variant("debug", default=False, description="Enable building with debug symbols")
-    variant("fortran", default=False, description="Build & install Fortran bindings")
+    variant("generic", default=False, description="Avoid specialization and larger code size")
+    variant("shared", default=True, description="Build shared library")
+    variant("debug", default=False, description="Enable debug symbols")
     variant(
         "tune",
         default="none",
         multi=False,
         values=TUNE_VARIANTS,
         description="Tune libint for use with the given package",
     )
-    variant(
-        "fma",
-        default=False,
-        description=(
-            "Generate code utilizing FMA (requires capable CPU and recent enough compiler)"
-        ),
-    )
+    variant("fma", default=True, description="Generate code utilizing FMA")
+    variant("cxx", default=False, description="Build and install the C++ API")
+    variant("eigen", default=False, description="Eigen used by implementation", when="+cxx")
+
+    description_fortran = "Build and install Fortran bindings"
+    variant("fortran", default=False, description=description_fortran)
+    for tune in TUNE_VARIANTS_CP2K:
+        variant("fortran", default=True, description=description_fortran, when=f"tune={tune}")
 
     # Build dependencies
-    depends_on("c", type="build")  # generated
+    depends_on("fortran", type="build", when="+fortran")
     depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("c", type="build")  # generated
 
     depends_on("autoconf@2.52:", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
     depends_on("python", type="build")
     depends_on("cmake@3.19:", when="@2.6.0:", type="build")
 
-    # Libint 2 dependencies
-    # Fixme: Can maintainers please confirm that this is a required dependency
-    depends_on(Boost.with_default_variants, when="@2:")
+    # Libint2 dependencies
+    # Boost is only needed if the Fortran interface is built
+    # A header-only installation (with no specific libraries)
+    # is sufficient (Boost.with_default_variants not needed)
+    depends_on("boost", when="@2: +fortran")
+    # Eigen is optional and not strictly necessary
+    depends_on("eigen", when="@2.7.0: +eigen")
```

Comment:
```suggestion
    depends_on("eigen", when="@2.7.0: +cxx")
```
? See comment above.

Reply:
Thank you, it's taken into the next (forced) commit.